components/Modal/Modal.tsx

```diff
@@ -52,12 +52,13 @@ const Modal: React.FC<IModalProps> = ({
       <ThemeVars theme="Modal" cssProps={styles} />
       <div className={styles.root}>
         <IconButton
+          title={!isOpen ? 'Open Share Menu' : 'Close Share Menu'}
```

Comment:
Modal component can be used for more than just the Share menu. This title can just be "Close".